Added Azure Application Gateway Ingress Controller (#17832)
kubernetes/website: content/en/docs/concepts/services-networking/ingress-controllers.md

## Additional controllers

> * [AKS Application Gateway Ingress Controller](https://github.com/Azure/application-gateway-kubernetes-ingress) is a fully supported ingress controller that enables ingress to [AKS clusters](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough-portal) using the [Azure Application Gateway](https://docs.microsoft.com/azure/application-gateway/overview).

Reviewer: ```suggestion
* [AKS Application Gateway Ingress Controller](https://github.com/Azure/application-gateway-kubernetes-ingress) is an ingress controller that enables ingress to [AKS clusters](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough-portal) using the [Azure Application Gateway](https://docs.microsoft.com/azure/application-gateway/overview).
```

I'd remove “fully supported” because “fully” is hard to define, and most other entries don't quantify how thorough the support is for different Ingress controllers.
